deploy/predict.py

```diff
@@ -32,7 +32,7 @@ def get_parser_infer(parents=None):
     parser.add_argument("--img_size", type=int, default=640, help="inference size (pixels)")
     parser.add_argument("--seed", type=int, default=2, help="set global seed")
 
-    parser.add_argument("--model_type", type=str, default="MindX", help="model type MindX/Lite")
+    parser.add_argument("--model_type", type=str, default="MindX", help="model type MindX/Lite/MindIR/ONNX")
```

Comment:
Please add usage methods and verified applicability in deploy/README.md(https://github.com/mindspore-lab/mindyolo/blob/master/deploy/README.md)

Reply:
DONE